theblues/terms.py

```diff
@@ -0,0 +1,53 @@
+from collections import namedtuple
+import datetime
+
+from theblues.errors import (
+    log,
+    ServerError,
+)
+from theblues.utils import (
+    ensure_trailing_slash,
+    make_request,
+)
+
+Term = namedtuple('Term', 'name title revision created_on content')
+
+
+class Terms(object):
+
+    def __init__(self, url):
+        """Initializer.
+
+        @param url The url to the Terms Service API.
+        """
+        self.url = ensure_trailing_slash(url)
+
+    def get_terms(self, name, revision=None):
+        """ Retrieve a specific term and condition.
+
+        @param name of the terms
+        @param revision of the terms
```

Comment:
Please mention that if revision is not provided the most recent revision is used.